src/goose/brokers/redis/batch.clj

```diff
@@ -1,15 +1,24 @@
 (ns goose.brokers.redis.batch
-  (:require [goose.brokers.redis.commands :as redis-cmds]
+  (:require [goose.batch :as batch]
+            [goose.brokers.redis.commands :as redis-cmds]
             [goose.defaults :as d]
             [goose.job :as job]
             [goose.retry]
             [goose.utils :as u]
-            [taoensso.carmine :as car]))
+            [taoensso.carmine :as car]
+            [taoensso.carmine.locks :as car-locks]))
 
+(def batch-state-keys [:id
```

Comment:
```suggestion
(def ^:private batch-state-keys [:id
```

Reply:
`batch-state-keys` is used in `batch_test` ns, this can't be made private